.eslintignore

```diff
@@ -4,6 +4,7 @@
 
 **/node_modules/**
 **/third_party/**
+/docs/**
```

Comment:
is there a pressing reason to do this? Ideally the example code should hit the same linting level as the rest of our code (we could put it an eslintrc that inherits but allows for console and whatever ([like we do for viewer](https://github.com/GoogleChrome/lighthouse/blob/7647d9ceea6649f3b44338d94435dbddbaa82a48/lighthouse-viewer/.eslintrc.js)) if things like that are the pain being avoided)

Reply:
done